File: FingoDriver/src/main/java/com/yelloco/fingodriver/models/networking/identify/IdentifyRequest.java
package com.yelloco.fingodriver.models.networking.identify;

public class IdentifyRequest
{
    public String verificationTemplate;

    public IdentifyRequest(){
    }

    public IdentifyRequest(String verificationTemplate) {
        this.verificationTemplate = verificationTemplate;
    }

    public String getVerificationTemplate() {
        return verificationTemplate;
    }

    public void setVerificationTemplate(String verificationTemplate) {
        this.verificationTemplate = verificationTemplate;
    }

    @Override
    public String toString() {
        return "IdentifyRequest{" +
                "verificationTemplate='" + verificationTemplate.substring(0, 10) + '\'' +
                '}';
    }
}

File: FingoDriver/src/main/java/com/yelloco/fingodriver/models/networking/identify/IdentifyRequest.kt
package com.yelloco.fingodriver.models.networking.identify

class IdentifyRequest
{
    var verificationTemplate: String? = null

    constructor() {}
    constructor(verificationTemplate: String?) {
        this.verificationTemplate = verificationTemplate
    }

    override fun toString(): String {
        return "IdentifyRequest{" +
                "verificationTemplate='" + verificationTemplate!!.substring(0, 10) + '\'' +
                '}'
    }
}